Tests/test_image_resample.py

```diff
@@ -268,10 +269,9 @@ def run_levels_case(self, i):
         px = i.load()
         for y in range(i.size[1]):
             used_colors = {px[x, y][0] for x in range(i.size[0])}
-            assert 256 == len(
-                used_colors
-            ), "All colors should present in resized image. Only {} on {} line.".format(
-                len(used_colors), y
+            assert 256 == len(used_colors), (
+                "All colors should present in resized image. "
```

Comment:
```suggestion
                "All colors should be present in resized image. "
```

Can also fix typo while doing this.